10.gün/script.js

```diff
@@ -116,59 +117,59 @@
 
 
 // Traversin the Dom (elementler üzerinde gezinme)
-let val;
-let list = document.querySelector(".list-group");
-val = list;
-val = list.childNodes;
+// let val;
```

Comment:
Let, const, var. Aralarindaki farki anladin mi? 
```suggestion
// let val = "add suggestion özelligini test ediyorum.";
```

Reply:
Evet let ile const farki o. Const sabit degerler icin yani. 
Var ve const/let arasinda da bir fark var.
Var olarak tanimladigin degiskene uygulamanin her yerinden erisebilirsin ancak let ve const ile tanimlanan degiskene sadece tanimladigin blok icinde erisirsin. Daha teknik terimlerle ifade edecek olursak var global scope da tanimlanir diger ikisi block scope da. 
Scope: kapsam, alan, faailet alani olarak cevrilebilir. Bir degisken veya fonksiyonun scope u dedigimizde onun nerelerde kullanilabilecegini belirtmis oluyoruz. Bu terim de teknik olarak ileride karsina siklikla cikacak. O yüzden alismakta fayda var. 
https://medium.com/kodcular/javascriptte-var-let-const-nedir-aralar%C4%B1nda-ki-farklar-nelerdir-3b1f6c9e23cd
Linkten ayrintili okuyabilirsin. Uzatmamak icin burada birakiyorum.

Reply:
Global olarak tanımladığımız bir değişkene blok içinde var, let, const ile erişiriz.  Ancak global olarak tanımlanan değişkenin değerini blok içinde "var" ile değiştirdiğimizde yada değeri değişecek bir işleme soktuğumuzda o değişkenin başta tanımlanan global değeri de değişir ki biz bunu istemeyiz.  ()bu durum let ve const için geçerli değil) 
Diğer taraftan Blok içinde "var" ile tanımladığımız değişkene blok dışından da erişiriz. (bu durum da let ve const için geçerli değil )